Minetest.md

```diff
@@ -0,0 +1,479 @@
+| K                             | Enable/disable fly mode (needs fly privilege)                  |
+| P                             | Enable/disable pitch move mode                                 |
+| J                             | Enable/disable fast mode (needs fast privilege)                |
+| H                             | Enable/disable noclip mode (needs noclip privilege)            |
+| E                             | Aux1 (Move fast in fast mode. Games may add special features)  |
+| C                             | Cycle through camera modes                                     |
+| V                             | Cycle through minimap modes                                    |
+| Shift + V                     | Change minimap orientation                                     |
+| F1                            | Hide/show HUD                                                  |
+| F2                            | Hide/show chat                                                 |
+| F3                            | Disable/enable fog                                             |
+| F4                            | Disable/enable camera update (Mapblocks are not updated anymore when disabled, disabled in release builds)  |
+| F5                            | Cycle through debug information screens                        |
+| F6                            | Cycle through profiler info screens                            |
+| F10                           | Show/hide console                                              |
+| F12                           | Take screenshot                                                |
+
+Paths
+-----
+Locations:
+
+* `bin`   - Compiled binaries
+* `share` - Distributed read-only data
+* `user`  - User-created modifiable data
+
+Where each location is on each platform:
+
+* Windows .zip / RUN_IN_PLACE source:
+    * `bin`   = `bin`
+    * `share` = `.`
+    * `user`  = `.`
+* Windows installed:
+    * `bin`   = `C:\Program Files\Minetest\bin (Depends on the install location)`
+    * `share` = `C:\Program Files\Minetest (Depends on the install location)`
+    * `user`  = `%APPDATA%\Minetest`
+* Linux installed:
+    * `bin`   = `/usr/bin`
+    * `share` = `/usr/share/minetest`
+    * `user`  = `~/.minetest`
+* macOS:
+    * `bin`   = `Contents/MacOS`
+    * `share` = `Contents/Resources`
+    * `user`  = `Contents/User OR ~/Library/Application Support/minetest`
+
+Worlds can be found as separate folders in: `user/worlds/`
+
+Configuration file
+------------------
+- Default location:
+    `user/minetest.conf`
+- This file is created by closing Minetest for the first time.
+- A specific file can be specified on the command line:
+    `--config <path-to-file>`
+- A run-in-place build will look for the configuration file in
+    `location_of_exe/../minetest.conf` and also `location_of_exe/../../minetest.conf`
+
+Command-line options
+--------------------
+- Use `--help`
+
+Compiling
```

Comment:
This part has been changed to document the process for dragonfire. It should stay in readme.md